lib/blockchain_double.js

```diff
@@ -583,6 +583,21 @@ BlockchainDouble.prototype.processBlock = function(vm, block, commit, callback)
 
       block.header.gasUsed = utils.toBuffer(to.hex(totalBlockGasUsage));
 
+      const txTrie = new Trie();
+      const rcptTrie = new Trie();
+      const promises = [];
+      const putInTrie = function(trie, key, val) {
+        return new Promise((resolve, reject) => {
+          try {
+            trie.put(key, val, (...args) => {
```

Comment:
`...args` seems unnecessary here. I think we should be handling an `error` param in this callback (with just `reject(error)`).

Reply:
actually... let's just use node's `promisify` on `trie.put` instead of wrapping it with `new Promise`. I don't think the `try`/`catch` is necessary as the `key` and `val` passed in will always be `Buffers` in this code, at least 2 bajillion tests will fail if someone comes in here and tries changes things.

Reply:
Agree 1000%